portal-api/api-tests/v1/gef/address.api-test.js

```diff
@@ -8,29 +8,74 @@ const { ordnanceSurvey } = require('../../../src/external-api/api');
 
 const { as, get } = require('../../api')(app);
 
-describe('GET /v1/gef/address/:postcode', () => {
-  const aPostcodeAddressUrl = `/v1/gef/address/${POSTCODE.VALID}`;
+jest.unmock('../../../src/external-api/api');
```

Comment:
same comment about this -- not sure i like this as our api-tests should only really care about our service (unless we're still mocking axios calls, which i dont think we are?)

Reply:
As above, I agree that this is not what we want (these API tests for Portal API shouldn't rely on MDM API working correctly to pass). As many of the API tests for Portal API work this way, I've left this for now and raised a ticket to address all of the instances of this issue in one go: https://ukef-dtfs.atlassian.net/browse/DTFS2-7194